bower.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "ng-oauth2",
+  "description": "An oauth2 implementation for angular based applications",
   "version": "0.0.6",
+  "hompage": "https://github.com/el-davo/ng-oauth2",
   "license": "MIT",
   "main": [
     "dist/ng-oauth2.min.js"
   ],
   "dependencies": {
-    "angular": "1.3.x",
-    "angular-mocks": "1.3.x",
+    "angular": "^1.3.x",
+    "angular-mocks": "^1.3.x",
```

Comment:
yeah - for some reason tests don't work over here.
It was originally not my intention to have all 4 changes in one pull request. will do a better job next time.